examples/angular/src/app/accordionWrapper/accordion.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, CUSTOM_ELEMENTS_SCHEMA, ElementRef, input } from '@angular/core';
+
+@Component({
+  selector: 'app-accordion',
+  standalone: true,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  templateUrl: './accordion.component.html',
+  styleUrls: ['./accordion.component.css'],
+})
+export class AccordionComponent {
+  constructor(elementRef: ElementRef) {
+    elementRef.nativeElement.addEventListener('ds-change', (event: Event) => {
+      console.log('Ds-change event fired', event);
+    });
+  }
+  bordered = input.required<string>();
+  items = input<{ title: string; content: string }[]>();
```

Comment:
I know this is just an example of what an app team _could_ do, but this component is capable of more than just having a string as content. It might be nice to show that complex content can go in there.

Reply:
I can beef it up.